ciab-assets/startup

```diff
@@ -45,6 +45,14 @@ def init_cluster():
     log("Configuring...")
     ctl("update-config", "--all", "--key", "minimum_core_count", "--value", "0")
     ctl("update-config", "--all", "--key", "minimum_memory_mb", "--value", "0")
+
+    http_api = os.getenv("HTTP_API", default=None)
+    http_api_port = os.getenv("HTTP_API_PORT", default="9000")
+    if http_api:
```

Comment:
Should this handle the case where `HTTP_API=OFF` or other values?

Reply:
If you feel strongly about it. Usually I follow the bash boolean env variable semantics of "non-empty is truthful"